package.json

```diff
@@ -83,6 +83,7 @@
     "core-js": "^2.6.11",
     "fs-extra": "^9.1.0",
     "highlight.js": "~9.18.1",
+    "lodash": "4.17.20",
```

Comment:
Could we not use lodash-es instead? It seems like we have 2 loadash dependencies now

Reply:
Updated to use lodash-es.